documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -37,7 +37,13 @@ spec:
 |Label |Description |Default value
 
 |`controller_max_vm_inflight`
-|The maximum number of VMs per plan that can be migrated simultaneously.
+a|*Varies with provider as follows*:
+
+* {rhv-full} migrations, {osp} migrations, VMware warm migrations, _or_ cold VMware migrations to a remote {ocp} cluster: The maximum number of disks that {project-short} can transfer simultaneously.
+
+* OVA migrations or cold VMware migrations using `virt-v2v` to a local {ocp} cluster: The maximum number of VMs that {project-short} can migrate simultaneously.
```

Comment:
```suggestion
* For {rhv-full} migrations, {osp} migrations to a remote {ocp} cluster: The maximum number of disks that {project-short} can transfer simultaneously.
* For OVA migrations using `virt-v2v` to a local {ocp} cluster: The maximum number of VMs that {project-short} can migrate simultaneously.
*  For VMware simultaneous migration:
** Cold migration: VMs for each ESXi host that can migrate simultaneously
** Warm migration: Disks for each ESXi host that can migrate simultaneously 
```

@rhoch @duduvaa - please can you have a look at this and see if it would be ok? i am trying to synthesize what has been approved with David's comments

Reply:
Thanks, @anarnold97 
@duduvaa Martin said the for VM cold migrations, there was a difference between local and remote. Do you see a difference in your tests?

Reply:
For cold remote we use the CNV CDI so the max in flight references the number of disks.

Reply:
@duduvaa can you check regarding Cold remote?